### PR TITLE
basic create class page for instructors.

### PR DESCRIPTION
**Pull request by apj11c**
Place holders for data, etc...

### CODE REVIEW

File: createClass.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		<input type="text" id="className" required>
+		<br>
+		Stream:
+		<select id="streamSelect" name="streamSelect" form="xcelform">
+			<option value="sample">SAMPLE</option>
+		</select>
+		<br>
+		Upload Xcel File:
+		<br>
+		<input type="file" id="xcelfile" accept=".xcel" required title=".xcel">
```
Comment (kabollinger03) [inline createClass.html]: Uuumm..... that is not a valid file extension. You want `.xlsx` and maybe `.xls`. See https://en.wikipedia.org/wiki/List_of_Microsoft_Office_filename_extensions

Comment (kabollinger03) [inline createClass.html]: Uuumm..... `.xcel` is not a valid file extension. You want `.xlsx` and maybe `.xls`. See https://en.wikipedia.org/wiki/List_of_Microsoft_Office_filename_extensions

**Review by kabollinger03 — CHANGES_REQUESTED** [review]

**Comment by kabollinger03** [comment]
The page title is `Create Class` but the heading is `Create Course`. That should be consistent. Should most likely be class because in the project vernacular "class" = "a group of students participating in a stream" and "course" = "a specific part of a module". Instructors do not have the "edit courses" permission.

While I'm at it, this page is also for editing information right? So shouldn't the title reflect that? IDK, maybe `create or edit class`?

**Review by kabollinger03 — APPROVED** [review]

**Review by ChrisSMendoza — APPROVED** [review]

**Comment by ChrisSMendoza** [comment]
Changes weren't being reflected in this pull request.

**Pull request closed without merging**